repose-aggregator/components/filters/rackspace-identity-basic-auth/src/main/scala/org/openrepose/filters/rackspaceidentitybasicauth/RackspaceIdentityBasicAuthHandler.scala

```diff
@@ -1,11 +1,13 @@
 package org.openrepose.filters.rackspaceidentitybasicauth
```

Comment:
i'm not sure about removing curlys on cases, it's just asking for bugs like leaving them off single line if's, unless there is something i'm forgetting?